Lib/test/test_pathlib.py

```diff
@@ -1412,6 +1413,8 @@ def cleanup():
             f.write(b"this is file B\n")
         with open(join('dirC', 'fileC'), 'wb') as f:
             f.write(b"this is file C\n")
+        with open(join('dirC', 'novel.txt'), 'w', encoding='ascii') as f:
+            f.write("this is a novel\n")
```

Comment:
You can just open the file in binary mode.

Reply:
done